solutions/observability/get-started/logs-essentials.md

```diff
@@ -0,0 +1,132 @@
+---
+navigation_title: Logs Essentials
+applies_to:
+  serverless: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+---
+
+# Get started with {{obs-serverless}} Logs Essentials [logs-essentials-get-started]
+
+```{note}
+Use this guide to get started with the Logs Essentials feature tier of {{obs-serverless}}. Refer to the main [{{observability}} get started](/get-started/index.md) docs to get started with {{obs-serverless}} Complete.
```

Comment:
```suggestion
Use this guide to get started with the Logs Essentials feature tier of {{obs-serverless}}. Refer to the main [{{observability}} get started](/solutions/observability/get-started/index.md) docs to get started with {{obs-serverless}} Complete, which includes APM and Infrastructure metrics.
```

Is "Complete" the official name of the full offering? If not, I would reword as "the complete Elastic Observability suite" or just "Elastic Observability".